__tests__/crawler.test.js

```diff
@@ -133,6 +133,10 @@ test("Crawl the docs and execute tests", async () => {
     'https://kb.isc.org/article/AA-01141/31/How-to-workaround-IPv6-prefix-length-issues-with-ISC-DHCP-clients.html',
     'https://www.snort.org/documents',
     'https://www.snort.org/downloads/#rule-downloads',
+    'https://www.projectcalico.org/everything-you-need-to-know-about-kubernetes-networking-on-azure/',
+    'https://www.tigera.io/calico-support/',
+    'https://www.tigera.io/legal/calico-cloud-support-policy',
+    'https://www.tigera.io/tutorials/?_sf_s=Deploy%20Service%20Mesh',
```

Comment:
I don't think we should be adding skips for these. If they don't work, they need to be fixed - on our side or their side.

Reply:
They all do work. They are just slow. I did a local build & test that passed. I pushed to PR and had these 4 dead links.